roles/satellite-clone/tasks/main.yml

```diff
@@ -190,17 +169,33 @@
   command: restorecon -R /
   when: restorecon
 
-# Run Satellite installer
-- name: run Satellite 6.1 installer
+# This file tells the candlepin puppet module in the Satellite installer that
+# candlepin is already set up. We remove it so candlepin is set up correctly.
+- name: Remove cpdb_done file
+  file:
+    path: /var/lib/candlepin/cpdb_done
+    state: absent
+
+- name: Run Satellite 6.1 installer
   command: katello-installer --capsule-dns false --capsule-dhcp false --capsule-tftp false
-  when: satellite_version == 6.1
-- name: run Satellite 6.2 installer
+  when: satellite_version == "6.1"
+
+- name: "Run Satellite 6.2 installer"
   command: satellite-installer --scenario satellite --foreman-proxy-dns false --foreman-proxy-dhcp false --foreman-proxy-tftp false --foreman-ipa-authentication false
-  when: satellite_version == 6.2
+  environment:
+    HOSTNAME: "{{ hostname }}"
+  when: satellite_version == "6.2"
+
+- name: "Run Satellite 6.3 installer"
+  command: satellite-installer --scenario satellite --foreman-proxy-dns false --foreman-proxy-dhcp false --foreman-proxy-tftp false --foreman-ipa-authentication false --disable-system-checks
```

Comment:
- what is the purpose of --disable-system checks?
- Is this applicable only for 6.3?

Reply:
Yes, I was confused about this as well, I guess it is not an option in 6.2 (I thought it was...) but the RAM check is going to block jenkins. I figure if someone has already installed and backed up Satellite on one machine (which they have if they are running clone), they understand the system requirements needed to run Satellite and we shouldn't be checking this as part of sat-clone and failing playbooks because of the system requirements check.